templates/content-widget-job_listing.php

```diff
@@ -14,21 +14,29 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
+
```

Comment:
Why are we adding a blank line here?

Reply:
I don't see that as being the end of the file from what I can see :)

Reply:
Fixed. That was not intended to be pushed with commit!